packages/@jsii/runtime/lib/program.ts

```diff
@@ -1,3 +1,5 @@
+import * as process from 'process';
```

Comment:
Added as part of my rumblings here... `process.env` is used here, and I kinda like to standardize on importing this even though it's implicit.